apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-1010-schema-changes/schema_diff.txt

```diff
@@ -0,0 +1,1588 @@
+CREATE TABLE public."Beneficiaries" (
```

Comment:
Apologies, but can you add a file header comment explaining what this is and why we need it?